src/PhpSpreadsheet/Collection/Cells.php

```diff
@@ -96,8 +96,61 @@ public function has($pCoord)
             return true;
         }
 
+        $column = '';
+        $row = 0;
+        sscanf($pCoord, '%[A-Z]%d', $column, $row);
```

Comment:
This might save some bytes, but sscanf adds some speed penalties. Es

Reply:
"few bytes" is an understatement. The logic in this file made `getSortedCoordinates` use _a lot_ less ram. In my case with a sheet of 1000 strings in the stringtable and some 350K cells did it go from 80MB to zero.